source/leadership.rst

```diff
@@ -0,0 +1,152 @@
+that the yt project can grow, nurture, and develop a healthy group of
+contributors, reviewers and maintainers of the project. Each color-coded group
+encompasses many members of the community, but hopefully will help to direct
+and distribute project member energy.
+
+**Red Group** (includes executive committee, any defined leadership roles) –
+Members of this group spend a large part of their time organizing the
+yt-project from a high level. They are expected to coordinate their actions
+with other red members so that the project presents a coherent position towards
+the outside. They are expected to prepare and attend the monthly team calls and
+executive committee meetings. We expect red members to teach and grow other
+members so that they can pass their organizational responsibilities on to them.
+This encourages us to increase diversity, reduce burn-out and allow group
+members to shift their focus in reaction to changing life circumstances. If a
+red member chooses to change focus, that change of focus will be announced at
+the next monthly team meeting. Red group members are expected to also perform
+duties of blue group members while facilitating and helping the blue group to
+grow.
+
+**Blue Group** (includes all yt project members) – This
+is where the journey of a new yt project team member starts. Blue group members
+spend a large part of your time facilitating contributions from others to the
+project (e.g. reviewing work, answering questions, maintaining project
+infrastructure, doing triage of new issues). We expect blue members to teach
+and grow contributors that can become blue members. This encourages diversity
+of the project. After building a good understanding of the technology and
+social structure of the project blue members can change to a red member by
+volunteering for tasks which a red member is expected to perform. After a while
+of doing so the group will ask if you want to become a red member and, if you
+agree, recognise you as such at a monthly team meeting.
+
+**Yellow Group** - Yellow group members are new contributors to the yt-project.
+Yellow group members do not have commit access to the yt project yet, but may
+have submitted a bugfix or documentation improvement and would like to
+contribute more. Yellow group members do not have high-level project
+responsibility, but may start to seek out involvement in the project by
+attending triage meetings. 
+
+**Green Group** (can be any person, any time, self-nominated only)-- This group
+applies to members of the community whose life situation has changed so that
+they prefer to (temporarily) not take on the rights and responsibilities of
+being an active member. Membership to the green group is invoked by
+self-nomination. They can return to active membership by resuming their
+community activities. Their return will be announced at the next team meeting
+or via the developers mailing list.
```

Comment:
Instead of "red", "blue", "yellow", and "green", I'd suggest "steering committee", "project members", "new contributors", and "temporary emeritus". If the red group isn't supposed to just be the steering committee I'd suggest calling it the "leadership group" or something like that, but I think it makes sense to align these groups with other existing governance structures.

I think having the group names be colors is confusing and I think the names I've proposed align with the descriptions of the color groups. I understand the desire to not add too much semantic meaning to the groups by naming them but I don't think that's worth the confusion they'll cause. 

I think formally having a way to declare yourself temporarily emeritus is a great idea as well as having a formal way of recognizing new contributors and people who want to participate in development.